giza/giza/content/options/tasks.py

```diff
@@ -44,11 +44,16 @@ def option_tasks(conf):
 
     tasks = []
     for dep_fn, option in o.content_iter():
-        filename = option.program.replace(' ', '-')
+        p = option.program.replace(' ', '-')
```

Comment:
Nor do I! Just wanted to make it work before I thought up sensible words :). Will update with 'program' and 'option' probably :)